main.cpp

```diff
@@ -127,393 +163,337 @@ struct CarWash
     int numPopulatedOutlets = 2;
 
     struct Outlet
     {
-        // which outlet number
         int outletNum = 1;
-        // current power
-        double currentPower = 0.0;
-        // GFCI status
-        std::string gfciState = "normal";
-        // Connector format
+        float currentPower = 0.0;
+        std::string GFCIState = "normal";
         std::string connectorFormat = "US";
-        // plug installed
         bool plugInstalled = false;
 
-        // Trip GFCI breaker
         void tripGFCI();
-        // retrieve outlet current power
-        double getCurrentPower();
-        // return plugged in status (useful for parent functions)
+        float getCurrentPower();
         bool isPlugInstalled();
-
     };
 
-    // Insert plug
+    Outlet outlet1;
+    Outlet outlet2;
+    Outlet outlet3;
+
     int insertPlug(Outlet outlet);
-    // get current outlet power
-    float getOutletPower(int plugNum);
-    // disable power
+    float getOutletPower();
     void disablePower();
-      
-
 };
-/*
-Thing 5) Pickups
-5 properties:
-    1) Magnet Material (std::string)
-    2) Number of Coils (int)
-    3) type of pickup
-    4) distance from strings in cm (float)
-    5) brand (std::string)
-3 things it can do:
-    1) adjust height
-    2) Select tap
-    3) convert vibration to voltage
- */
+
+void PowerStrip::Outlet::tripGFCI()
+        {
+            currentPower = 0.0;
+            GFCIState = "tripped";
+        }
+float PowerStrip::Outlet::getCurrentPower() { return currentPower;}
+bool PowerStrip::Outlet::isPlugInstalled() { return plugInstalled;}
+
+int PowerStrip::insertPlug(Outlet outlet) 
+{
+    outlet.plugInstalled = true;
+    return outlet.outletNum;
+}
+
+float PowerStrip::getOutletPower()
+{
+    return outlet1.getCurrentPower() + outlet2.getCurrentPower() + 
+            outlet3.getCurrentPower();
+}
+
+void PowerStrip::disablePower()
+{
+    outlet1.GFCIState = "off";
+    outlet2.GFCIState = "off";
+    outlet3.GFCIState = "off";
+}
+
 struct Pickup
 {
-    // Magnet Material 
     std::string magnetMaterial = "alnico";
-    // Number of Coils 
     int numCoils = 2;
-    // type of pickup 
     std::string pickupType = "Humbucker";
-    // distance from strings in cm 
-    float pickupDistance = 3.2f;
-    // brand 
+    double pickupDistance = 3.2;
     std::string brand = "EMG";
     
-    // adjust height in mm
     void adjustHeight(double heightAdjustment = 0.0);
-    // Select tap
     int toggleCenterTap();
-    // convert vibration to voltage
     double vibToVoltage(double vibrationPower = 0.0);
 };
-/*
-Thing 6) Neck
-5 properties:
-    1) Fretboard Material (std::string)
-    2) scale length in cm (double)
-    3) number of frets(int)
-    4) fret width in mm (float)
-    5) nut material (std::string)
-3 things it can do:
-    1) adjust truss rod
-    2) fret note
-    3) clean fretboard
- */
+
+void Pickup::adjustHeight(double heightAdjustment){ pickupDistance -= heightAdjustment;}
```

Comment:
If you write single-line functions, make sure you have space around the `{ }` so the code is easier to read
- [ ] revise